src/content/blog/whats-new-february-2024.mdx

```diff
@@ -0,0 +1,195 @@
+  { url: 'https://warawul.coffee/en/', title: '@chris_origins_coffee'},
+  { url: 'https://lindakat-arcade.netlify.app/', title: '@lindakat'},
+  { url: 'https://daniell.dev/', title: '@daniellwdb'},
+  { url: 'https://heschl.dev/', title: '@joneswastaken'},
+  { url: 'https://www.benjamindorey.fr/', title: '@biommxx'},
+  { url: 'https://pro-g.eu/', title: '@biommxx'},
+  { url: 'https://hitesh.is-a.dev/', title: '@hitesh_dev'},
+  { url: 'https://www.tekac.club/humans', title: '@emptybrother7'},
+  { url: 'https://evadecker.com/design', title: '@for.eva.new'},
+  { url: 'https://joelmturner.com/', title: '@joelmturner'},
+  { url: 'https://hermes.cadens.studio/', title: '@cadens.studio'},
+  { url: 'https://barvian.me/', title: '@mbarvian'},
+  { url: 'https://www.deskpro.com/', title: '@mattheous'},
+  { url: 'https://wybm-valentine.netlify.app/', title: '@withered.flowers'},
+  { url: 'https://mike-astrotutorial.netlify.app/', title: '@mikeoxygen'},
+  { url: 'https://www.thatisrich.co.uk/', title: '@thatisrich'},
+  { url: 'https://wallcreo.com/', title: '@stan.kukucka'},
+  { url: 'https://osteopathe-le-thor.fr/', title: '@arnaud_83976'},
+  { url: 'https://dan-portoflio-v3.vercel.app/', title: '@rythmlord'},
+  { url: 'https://www.xypnox.com/blag/posts/themescura/introducing-themescura/', title: '@xypnox'},
+  { url: 'https://photosite-3cw.pages.dev/', title: '@casungo'},
+  { url: 'https://sanquirinoestudio.com.ar/', title: '@.wateredplant'},
+  { url: 'https://suplovers.co/', title: '@alejostereo'},
+  { url: 'https://gitarchived.org/', title: '@thebigbot'},
+  { url: 'https://www.arjancodes.com/', title: '@santiago6916'},
+  { url: 'https://pdx-food-weeks.vercel.app/', title: '@joelmturner'},
+  { url: 'https://www.helmerdavila.com/', title: '@dynamo_dev_99'},
+  { url: 'https://lakelimbo.github.io/', title: '@lakelimbo'},
+  { url: 'https://aperofrancophone.com/', title: '@marvinkr'},
+  { url: 'https://www.karelvo.com/', title: '@karel5743'},
+  { url: 'https://anuwong.com/', title: '@anuwong'},
+  { url: 'https://k16e.co/', title: '@k16e.co'},
+  { url: 'https://wordclues.com/', title: '@plinpod'},
+  { url: 'https://gitter.im/'},
+  { url: 'https://freestuff.dev/'},
+  { url: 'https://overlayed.dev/', title: '@hacksore'},
+  { url: 'https://5pin-talk.ca/', title: '@jbrains_'},
+  { url: 'https://upshot.dev/', title: '@khaya_zulu'},
+  { url: 'https://prodkt.cloud/', title: '@prodkt'},
+  { url: 'https://soulrest.melos.church/', title: '@cameronpak'},
+  { url: 'https://designzen.co/', title: '@frank_stallone'},
+  { url: 'https://newsletter.howd.dev/posts/01-gus-moxie/', title: '@bonniedotdev'},
+  { url: 'https://www.mickeymarse.dev/', title: '@mickeymarse'},
+  { url: 'https://mingjun.lu/blog', title: '@mingjunlu'},
+  { url: 'https://coderdiaz.com/', title: '@coderdiaz'},
+  { url: 'https://streamdal.com/', title: '@dosplatos'},
+  { url: 'https://moyadaniel.com/', title: '@rythmlord'},
+];
```

Comment:
I think these are the new ones so far:

for.eva.new stuff from Tuesday:
https://eva.town/projects/boundaries-map
https://eva.town/projects/commonplace

 https://futrloggr.vercel.app/ by xylis_

https://loquierasono.democracia.com.es/ by aitor